templates/parts/actor/ffg-talents.html

```diff
@@ -26,5 +25,20 @@
       </div>
     </li>
     {{/each}}
+    {{#each data.talentList as |item id|}}
+    <li class="item specialization-talent-item flexrow" data-item-id="{{item.itemId}}">
+      <div class="item-name">
+        {{item.name}}
+      </div>
+      <div>{{localize item.activationLabel}}</div>
```

Comment:
item.activationLabel just ends up being "Activation". It should just be item.activation.

Reply:
good catch, fixed